openstack/identity/v3/applicationcredentials/results.go

```diff
@@ -39,8 +53,8 @@ type ApplicationCredential struct {
 	Roles []Role `json:"roles"`
 	// The expiration time of the application credential, if one was specified.
 	ExpiresAt time.Time `json:"-"`
-	// Links contains referencing links to the application credential.
-	Links map[string]interface{} `json:"links"`
```

Comment:
Let's add this back. I mentioned that no one has noticed when these fields are missing, but I'd prefer to leave the existing ones in place just in case they're being used somewhere.